setup.py

```diff
@@ -145,4 +107,4 @@ def build_json_table() -> int:
     return result.returncode
 
 
-atexit.register(build_json_table)
+# atexit.register(build_json_table)  ## does not work with build isolation
```

Comment:
In setup we need to build some tables.

Can this be done in that dynamic section like it is for "version"?

Reply:
> In setup we need to build some tables.

If users download the sdist (source tarball) from PyPI, would the tables be included there, or would they be built on the user's system?

Reply:
> > In setup we need to build some tables.
> 
> If users download the sdist (source tarball) from PyPI, would the tables be included there, or would they be built on the user's system?

Tables are included in PyPI tarballs, eggs, and wheel. This is done via https://github.com/Mathics3/mathics-scanner/blob/master/setup.py#L102-L104 along with Makefile targets which make sure files are around in development.

Reply:
> I think this was _also_ added to `setup.py` as a way to generate tables as well

Yes, that's the one that I had to comment out.

I've just pushed a change ([52fbe3d](https://github.com/Mathics3/mathics-scanner/pull/58/commits/52fbe3d38e0e1c9d0546269685b9b0db65842cde) that does this in the `egg_info` phase of building an sdist (or wheel)